phpunit.xml.dist

```diff
@@ -1,23 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
+         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/10.5/phpunit.xsd"
          backupGlobals="false"
          colors="true"
-         convertDeprecationsToExceptions="false"
+         cacheDirectory=".phpunit.cache"
```

Comment:
is this not default or something?

Reply:
According to the documentation there is no default param for this one https://docs.phpunit.de/en/11.3/configuration.html#the-cachedirectory-attribute.